external/libtuv/source/tinyara/uv_tinyara_signal.c

```diff
@@ -131,7 +129,7 @@ static uv_signal_t* uv__signal_first_handle(int signum) {
   lookup.signum = signum;
   lookup.loop = NULL;
 
-  index = PID_HASH(getpid());
+  index = PIDHASH(getpid());
   if (index < 0 || index >= CONFIG_MAX_TASKS) {
```

Comment:
index is 0 or more and less than 'CONFIG_MAX_TASKS'. I wonder why check index.

Reply:
Yes, you are right. But I think it is good for checking the validation for index.

Reply:
Could you let me know whether it is a valid checking or not?